### PR TITLE
Making a Gitbook out of the T2 docs repo

### PR DESCRIPTION
**Pull request by Frijol**
working towards https://github.com/tessel/t2-docs/issues/50

### CODE REVIEW

**Comment by Frijol** [comment]
cc @reconbot and @tcr you are both likely to have opinions on this project

**Comment by Frijol** [comment]
removed WIP flag because technically this is already better than tessel.io/docs (consistent syntax highlighting, for example).

However, we would need to figure out how we want to use the gitbook version in deployment - is it possible to map it to tessel.io/docs? (maybe through their enterprise version?)

**Comment by Frijol** [comment]
screen shot 2016-04-13 at 4 30 38 pm

**Comment by johnnyman727** [comment]
> However, we would need to figure out how we want to use the gitbook version in deployment - is it possible to map it to tessel.io/docs? (maybe through their enterprise version?)

http://til.secretgeek.net/gitbook/host_on_custom_subdomain.html

**Comment by tcr** [comment]
@frijol This LGTM and should be ready to merge.

We have two approaches here:

1. Rehost content on tessel.io/docs much like our blog by tunneling requests through (for example) S3.
2. Host content directly from S3 on http://docs.tessel.io/ and redirect tessel.io/docs to that.

Either is pretty easy and we should just pull the trigger on one. #2 is slightly easier but breaks old links, I suppose.

**Comment by Frijol** [comment]
@tcr I forget where we left this. Are you working on a docs.tessel.io?

**Comment by Frijol** [comment]
just realized the best thing about gitbooks is that it lets us have a glossary, which should do wonders for our accessibility :) (cc @HipsterBrown )

**Comment by HipsterBrown** [comment]
@Frijol How can a view this locally?

**Comment by Frijol** [comment]
@HipsterBrown should say in the PR'd version of the README. Try it out and let me know if the instructions are good?

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 
-![](https://raw.githubusercontent.com/rwaldron/tessel-io/master/fritzing/tessel.png)
+* [gitbook](https://www.npmjs.com/package/gitbook)
+* [gitbook-summary](https://github.com/imfly/gitbook-summary)
 
-[Fritzing part for Tessel](https://github.com/rwaldron/tessel-io/tree/master/fritzing)
+# Running locally
 
-**Get involved:** The Tessel project welcomes contributors! Get started [here](https://github.com/tessel/project) and reach out if you have any questions.
+Clone, `npm install`, and `gitbook serve`
```
Comment (HipsterBrown) [inline README.md]: There is no `package.json` in this repo, so `npm install` fails. There are two options for changing this:

- add a `package.json` with the `gitbook-cli` devDependency
- change this to say "Close, `npm install -g gitbook`, and `gitbook serve`

I think the less global tooling of contributors to install, the better.

Comment (Frijol) [inline README.md]: To be clear, you would prefer the package.json option?

Comment (HipsterBrown) [inline README.md]: Yup. I'll make a PR soon with some setup updates overall.

**Comment by Frijol** [comment]
@HipsterBrown are we now in a good place to deploy?

**Comment by HipsterBrown** [comment]
@Frijol LGTM :+1:

Pinging @reconbot for some ideas of how to deploy this.

**Comment by Frijol** [comment]
@reconbot any suggestions? Really want to get this online, but don't know how to do it.

**Comment by Frijol** [comment]
Great! Will do, and I've just messaged Gitbooks asking for a free "organization" account (which they give to open source projects)

**Comment by Frijol** [comment]
cool, we have https://www.gitbook.com/@tessel now

**Comment by HipsterBrown** [comment]
Thanks @reconbot 🎉 

If we merge this PR, will the hosted book auto-update?

**Comment by Frijol** [comment]
nearly just merged this but realized it would kill the current docs page because it pulls from links which are changed here. Pends https://github.com/tessel/tessel.io/issues/96 resolution

**Comment by Frijol** [comment]
URL is https://tessel.gitbooks.io/t2-docs/content/, which is what I expected. Are you talking about something else?

**Pull request merged**